modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -103,6 +103,8 @@ class SeqexecEngine(settings: SeqexecEngine.Settings) {
             // TODO: Implement skipping at Engine level
             skip = false,
             configStatus = Map.empty,
+            // Where do I get the file id from?
+            None,
```

Comment:
This is a key part of the engine update. We need to find how to populate this field from the DHS generated value